tools/browsers-lists/utils.mjs

```diff
@@ -16,8 +16,8 @@ export function getBrowserName ({ browserName, platformName }) {
 }
 
 export function getBrowserVersion (capabilities) {
-  if (capabilities.platformName?.toLowerCase() === 'ios') {
-    return capabilities['appium:platformVersion'] || capabilities.platformVersion
+  if (['ios', 'android'].includes(capabilities.platformName?.toLowerCase())) {
+    return capabilities['appium:platformVersion'] || capabilities.version
```

Comment:
Why was this changed?

Reply:
Matcher wasn't respecting versioning for Android. One of the currently failing soft nav test needs to be excluded from Android 9, but `'android>9'` not working atm because of this.